.github/workflows/regression-report.yml

```diff
@@ -454,10 +454,30 @@ jobs:
 
       - name: Prepare environment
         run: |
-          mv .ci-temp/project.properties ./.ci-temp/contribution/checkstyle-tester/
-          mv .ci-temp/*.xml ./.ci-temp/contribution/checkstyle-tester/
+          mv .ci-temp/project.properties ./.ci-temp/test-configs/checkstyle-tester/
+          mv .ci-temp/*.xml ./.ci-temp/test-configs/checkstyle-tester/
           sudo apt install groovy
 
+      - name: Download DiffTool JAR
+        uses: robinraju/release-downloader@v1.8
+        with:
+          repository: "relentless-pursuit/test-configs"
+          tag: "diff-tool-1.0.0"
+          fileName: "difftool.jar"
```

Comment:
Reminder for me:
Do release of this to org repository

Reply:
done: https://github.com/checkstyle/test-configs/releases/tag/diff-java-tool-1.0-SNAPSHOT-all
compiled by jdk21

Reply:
updated